### ISSUE TITLE
Deep dependency in webworkers could not be solved correctly

### ISSUE THREAD

**Issue by mountain** [requirejs/require-cs#11]
Check with below demo and the code:
-  http://playground.wahlque.org/tutorials/coffeescript/004-webworkers/
-  https://github.com/Wahlque/tutorials/blob/master/coffeescript/004-webworkers/multiplication.coffee#L17
-  https://github.com/Wahlque/tutorials/blob/master/coffeescript/004-webworkers/multiplication.coffee#L23

From line 17 to line 23, I add cases one by one to increase the complexity, and the cases failed from the fifth.

In summary, 
- All modules are downloaded by browser 
- If a module referenced by the first layer and the module do not dependent with others,  it will be work.
- If a module referenced by the first layer and the module dependent with others, the other module will be a null value in the scope, and then the program failed

It might be some kind of scoping problem, I think.
